wrapper.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
We need this little wrapper script to start our `flags/change-flag.sh` script in the background, and to forward signals to flagd so we can gracefully shut down.